src/status_im/i18n.cljs

```diff
@@ -52,73 +53,163 @@
+                             :fy          fy/translations
+                             :he          he/translations
+                             :hi          hi/translations
+                             :hu          hu/translations
+                             :id          id/translations
+                             :it          it/translations
+                             :it-ch       it-ch/translations
+                             :ja          ja/translations
+                             :ko          ko/translations
+                             :la          la/translations
+                             :lt          lt/translations
+                             :lv          lv/translations
+                             :ms          ms/translations
+                             :nb          nb/translations
+                             :ne          ne/translations
+                             :nl          nl/translations
+                             :pl          pl/translations
+                             :pt-br       pt-br/translations
+                             :pt-pt       pt-pt/translations
+                             :ro          ro/translations
+                             :ru          ru/translations
+                             :sl          sl/translations
+                             :sr          sr-rs-cyrl/translations
+                             :sr-RS_#Latn sr-rs-latn/translations
+                             :sr-RS_#Cyrl sr-rs-cyrl/translations
+                             :sv          sv/translations
+                             :sw          sw/translations
+                             :th          th/translations
+                             :tr          tr/translations
+                             :uk          uk/translations
+                             :ur          ur/translations
+                             :vi          vi/translations
+                             :zh          zh-hans/translations
+                             :zh-hans     zh-hans/translations
+                             :zh-hans-tw  zh-hans/translations
+                             :zh-hans-sg  zh-hans/translations
+                             :zh-hans-hk  zh-hans/translations
+                             :zh-hans-cn  zh-hans/translations
+                             :zh-hans-mo  zh-hans/translations
+                             :zh-hant     zh-hant/translations
+                             :zh-hant-tw  zh-hant/translations
+                             :zh-hant-sg  zh-hant/translations
+                             :zh-hant-hk  zh-hant/translations
+                             :zh-hant-cn  zh-hant/translations
+                             :zh-hant-mo  zh-hant/translations
+                             :zh-wuu      zh-wuu/translations
+                             :zh-yue      zh-yue/translations})
+
+;; english as source of truth
+(def trans-ids (set (keys en/translations)))
```

Comment:
we call them labels in the app afaik so it would be better to keep the name

Reply:
Renamed.